gradio/static/js/all_io.js

```diff
@@ -56,12 +56,12 @@ var io_master_template = {
       this.target.find(".output_interfaces").css("opacity", 1);
     }
   },
-  flag: function(message) {
+  flag: function() {
```

Comment:
Can we just remove `message` if we're not using it?

Reply:
There's no message anymore, this change was to remove it. See above for msg/message confusion.